app/controllers/submissions_controller.rb

```diff
@@ -0,0 +1,24 @@
+# SubmissionsController
```

Comment:
I think we have a SubmissionController already, we should probably rename it to plural, and add this in.

That's my bad I named it wrong